packages/fiori/src/ViewSettingsDialog.js

```diff
@@ -157,7 +157,7 @@ const metadata = {
 		 *
 		 * @event sap.ui.webcomponents.fiori.ViewSettingsDialog#cancel
 		 * @param {String} sortOrder The current sort order selected.
-		 * @param {String} sortBy The current sort by selected.
+		 * @param {String} sortBy The current sort element selected.
```

Comment:
Could be written more explicitly like so:
sortBy The currently selected <code>ui5-sort-item</code> text property